spec/factories/generic_works.rb

```diff
@@ -77,7 +77,10 @@
       end
     end
 
-
-
+    # a public work with complete metadata and a real image! Slow.
```

Comment:
If you'd like it to be faster, you could do an option without any files at all. there are plenty of metadata and relationship-y things you can mess around with using works alone, and our implementation doesn't require a work to have a file attached to it (whereas default sufia does).